secp256kfun/src/hash.rs

```diff
@@ -150,6 +150,15 @@ impl<K: HashInto, V: HashInto> HashInto for alloc::collections::BTreeMap<K, V> {
     }
 }
 
+#[cfg(feature = "alloc")]
+impl<T: HashInto + Ord> HashInto for alloc::collections::BTreeSet<T> {
+    fn hash_into(self, hash: &mut impl digest::Update) {
+        for item in self {
```

Comment:
Will this item ordering be consistent with other implementations?
Maybe a slice of party indices would be better?

Reply:
Consistent with other implementations of hashing BTreeSets? I mean there's only one way to hash an ordered set...
With respect to FROST signing I don't see any reason why anyone would order things any differently or why we should worry about that now.

Reply:
I was more-so thinking for future compatibility with other implementations that might not use BTreeSets (python). This ordering on each signer will need to match to produce a valid signature. But I forgot that BTreeSet uses the underlying Ord, so the underlying sorting here likely already makes sense.